### PR TITLE
Handle redirected stdout on Windows

### PR DESCRIPTION
**Pull request by fire**
1. Handle redirected tty.
2. Build on master.

### CODE REVIEW

File: lib/godot/_internal/utils/doc_utils.py [hunk 1]
```diff
@@ -53,40 +53,60 @@ def _remove_sgr(text: str) -> str:
...
 
-	replacements = _replacements_for_tty if for_tty else _replacements
+    "gdscript": "",
+    "_gdscript": "",
 
-	def replace(match_):
-		groups = match_.groupdict()
+    "csharp": "",
+    "_csharp": "",
+}
```
Comment (fire) [inline lib/godot/_internal/utils/doc_utils.py:77]: Delete due to redundancy.

**Comment by fire** [comment]
Need to test with Windows 4.2 on steam.

**Comment by fire** [comment]
Test plan.

Go to the test project. 

On `Spawner` node path change the spawn_count from ??? to 1.

If it works then the python extension works.

**Comment by fire** [comment]
Works.

**Comment by maiself** [comment]
Hadn't realized that `stdout` et al could be `None`, might be a platform specific thing? I wonder if there are other places where this could be a problem.

In any case, this looks good to me, thanks!

**Pull request merged**